doc/user/cluster_tls.md

```diff
@@ -54,9 +54,9 @@ Once passed, etcd-operator will mount this secret at `/etc/etcdtls/member/peer-t
 
 The client TLS assets should have the following:
 - **server.crt**: etcd server's client communication cert.
-  The certificate should allow wildcard domain `*.${clusterName}.${namespace}.svc.cluster.local`,
-  `${clusterName}-client.${namespace}.svc.cluster.local`, `localhost`.
-  In this case, it is `*.example.default.svc.cluster.local`, `example-client.default.svc.cluster.local`, and `localhost`.
+  The certificate should allow wildcard domain `*.${clusterName}.${namespace}.svc`,
```

Comment:
mentioning that users also need to provide whatever DNS they might use to contact the etcd members. especially, when they use a different cluster domain.